src/routes.js

```diff
@@ -44,4 +45,20 @@ router.post('/coffeegrams', async ctx => {
   ctx.redirect(`/users/${ctx.state.currentUser.username}`);
 });
 
+router.post('/users/:id', async ctx => {
+  var { bio } = ctx.request.body;
+  if (ctx.params.id !== ctx.state.currentUser.id) {
+    ctx.request.flash('error', 'You\'re not allowed to edit someone else\'s profile');
+    return ctx.redirect('back');
+  }
+  if (isBlank(bio)) {
+    ctx.request.flash('error', 'Your bio looks empty!');
+    return ctx.redirect('back');
+  }
+  var id = ctx.params.id;
```

Comment:
This isn't actually an ID, it's the user's username ;) as in, `manuscriptmastr` in `/users/manuscriptmastr`